src/components/MemoryBoxView.tsx

```diff
@@ -9,12 +9,14 @@ import Card from 'react-bootstrap/Card';
 import Popover from 'react-bootstrap/Popover';
 import PropTypes from 'prop-types';
 
+import Canvas from './Canvas';
 import DeviceInfoView from './DeviceInfoView';
 import MemoryView from './MemoryView';
 
 interface MemoryBoxViewProps {
     title: string;
     description?: string;
+    hasViz?: boolean;
```

Comment:
Could you name this parameter to something more descriptive. I don't quite understand what it's function is.

Reply:
renamed hasViz to containsVisualization.